specs/v1.yaml

```diff
@@ -1651,6 +1970,100 @@ components:
                   This is useful for nonprofits to indicate when they have received and processed the grant.
                   Expressed in ISO 8601 format.
                   A null value indicates that the grant has not been acknowledged.
+    RecurringGrantCaptureRequest:
+      description: |-
+        The request to create and submit a monthly recurring grant. This is useful to capture a recurring grant intent associated with Connect workflow session.
+        The request should specify the grant amount, start date and an (optional) end date. If end date is not specified, the recurring grant will continue indefinitely. 
+        These are the amount and dates submitted for processing by the DAF.
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - workflowSessionId
+              - amount
+              - startDate
```

Comment:
startDate should not be required IMO; if not provided it's assumed we start today/now